styles.css

```diff
@@ -0,0 +1,18 @@
+body {
```

Comment:
this is a particular case because the body starts with a margin 8px defined by the browser. thats when you should use a reset file like normalize.css  or reset.css 
you can check this here: http://meyerweb.com/eric/tools/css/reset/

please remove this selector when you add the reset or normalize file.